wire-ios-data-model/Source/Model/User/Availability/Availability.swift

```diff
@@ -19,4 +19,8 @@
 @objc(ZMAvailability)
 public enum Availability: Int, CaseIterable {
     case none, available, busy, away
+
+    public init() {
+        self = .none
+    }
```

Comment:
To read this from some place, I find this more implicit and one needs to look up what `= AccentColor()` case is chosen compared to `= .none`.

Reply:
I wanted to offer an initialiser like any other primitive has, e.g. `Bool()`, `String()`, `Int()` etc.
But I'll revert it.